content/adventures/nl.yaml

```diff
@@ -368,15 +368,15 @@ adventures:
                     {while} antwoord != 25
                         antwoord = {ask} 'Wat is 5 keer 5?'
                     {print} 'Goed gedaan!'
-                story_text: |
+                story_text: |-
                     ## Meer lussen!
                     Nu gaan we een nieuwe loop leren: De `{while}` loop!
                     We gaan door zo lang het statement dat we hebben waar is.
                     Vergeet dus niet in de loop de waarde te veranderen.
 
                     Dus bij de voorbeeldcode gaan we door totdat er een goed antwoord is gegeven.
                     Als er nooit een goed antwoord wordt gegeven, dan stopt de loop nooit!
-                example_code: "## Example Hedy code\n```\nanswer = 0\nwhile answer != 25\n    answer = ask 'What is 5 times 5?'\nprint 'A correct answer has been given'\n```\n"
+                example_code: "## Example Hedy code\n```\nanswer = 0\n{while} answer != 25\n    answer = {ask} 'What is 5 times 5?'\n{print} 'A correct answer has been given'\n```\n"
```

Comment:
Not sure why the placeholders are fixed but the example code is still in English?

Reply:
Because someone fixed it while translating NL? Not the smartes way indeed, but OK I think?